var/spack/repos/builtin/packages/py-tf-keras/package.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyTfKeras(PythonPackage):
+    """The TF-Keras library is a pure TensorFlow implementation of Keras,
+    based on the legacy tf.keras codebase. Note that the "main" version
+    of Keras is now Keras 3 (formerly Keras Core), which is a
+    multi-backend implementation of Keras, supporting JAX, PyTorch, and TensorFlow.
+     Keras 3 is being developed at keras-team/keras."""
+
+    homepage = "https://github.com/keras-team/tf-keras"
+    pypi = "tf_keras/tf_keras-2.16.0.tar.gz"
+
+    maintainers("jonas-eschle")
+
+    license("Apache-2.0", checked_by="jonas-eschle")
+
+    max_minor = 16
+    version("2.16.0", sha256="db53891f1ac98197c2acced98cdca8c06ba8255655a6cb7eb95ed49676118280")
+
+    # Supported Python versions listed in multiple places:
+    # * tf-keras/tools/pip_package/setup.py
+    # * CONTRIBUTING.md
+    # * PKG-INFO
+    depends_on("python@3.9:", type=("build", "run"), when="@2.16:")
+    depends_on("py-setuptools", type="build")
+
+    # Required dependencies listed in multiple places:
+    # * BUILD
+    # * WORKSPACE
+    depends_on("py-absl-py", type=("build", "run"))
+    depends_on("py-h5py", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-pandas", type=("build", "run"))
+    depends_on("pil", type=("build", "run"))
+    depends_on("py-portpicker", type=("build", "run"))
+    depends_on("py-pydot", type=("build", "run"))
+    depends_on("py-scipy", type=("build", "run"))
+    depends_on("py-six", type=("build", "run"))
+    # the tf-keras versions are following along with TF versions
+    for minor_ver in range(16, max_minor + 1):
+        depends_on(
+            "py-tensorflow@2.{}".format(minor_ver),
+            type=("build", "run"),
+            when="@2.{}".format(minor_ver),
+        )
+        depends_on(
+            "py-tensorboard@2.{}".format(minor_ver),
+            type=("build", "run"),
+            when="@2.{}".format(minor_ver),
+        )
+    depends_on("py-pyyaml", type=("build", "run"))
+    depends_on("bazel", type="build")
+    depends_on("protobuf", type="build")
+
+    @when("@2.16:")
```

Comment:
when is not necessary, this package only has one version

Reply:
was for convenience, forward compat, but removed